advanced-features/prompt-playground/setup.mdx

```diff
@@ -40,26 +40,29 @@ settings = {
 async def call_llm():
     generation = cl.ChatGeneration(
         provider=ChatOpenAI.id,
-        inputs=inputs,
+        variables=variables,
         settings=settings,
         messages=[
             cl.GenerationMessage(
-                template=template, formatted=template.format(**inputs), role="assistant"
+                content=template.format(**variables), role="user"
```

Comment:
cl.GenerationMessage class no longer exists. It is now a dict. It makes examples with openai much easier since you can use the exact same dicts for both the generation and the openai call